tensorflow_addons/utils/test_utils.py

```diff
@@ -26,6 +26,7 @@
 from tensorflow.python.framework.test_util import run_all_in_graph_and_eager_modes
 from tensorflow.python.framework.test_util import run_deprecated_v1
 from tensorflow.python.framework.test_util import run_in_graph_and_eager_modes
+from tensorflow.python.framework.test_util import use_gpu
```

Comment:
This seems fairly straightforward to just fork?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/framework/test_util.py#L1420

Reply:
done :-)